internal/wire/op_msg.go

```diff
@@ -100,6 +100,8 @@ func (msg *OpMsg) Document() (*types.Document, error) {
 	for _, doc := range docs {
 		values := doc.Values()
 
+		// TODO move that check to validateValue?
```

Comment:
Why `?` :)

Reply:
Because I'm not sure: maybe we should do that, maybe something else, maybe  we should move everything to some other place